README.md

```diff
@@ -1,44 +1,243 @@
-# The IDS-BaseCamp
+**Mission for Launch Coalition 2022**
 
-Similar to the [Testbed](https://github.com/International-Data-Spaces-Association/IDS-testbed), the IDS-BaseCamp provides developers functioning components with just enough features to be usable for a secure and sovereign data exchange, yet its intended purpose is to enable developers to experiment with their own ideas and implement their visions, as out-of-pocket as they may be, in it, without any regard to the standardized specifications of the IDSA. This playground for ideas can be used before trying out a developers idea to the Testbed, in which it would have to adhere to certain guidelines. 
+_ **Let's build a base camp for data spaces** _
 
-Contrary to the [MVDS](https://github.com/International-Data-Spaces-Association/IDS-testbed/blob/master/minimum-viable-data-space/MVDS.md), the IDS-BaseCamp is not confined by the criteria of a minimal working state, it not only can, but is supposed to develop and explore new branches and possibilities of a dataspace.
+Data Spaces built according to the IDSA specifications are growing in numbers and complexity. The Basecamp initiative is working towards the creation of an integrated and tested set of OSS components which can be used in pilot as well as in productive environments, leveraging the experience and from contributions of projects in all stages of the lifecycle.
 
-The ISD-BaseCamp is the unique solution provided by IDSA Head Office, who will function as moderators.
+The goal is to enable developers, testbeds and productive systems to be able to work on a common basis and leverage the power of many to constantly improve the code set whilst maintaining the highest level of interoperability. At the same time the efforts of project teams can shift from setup and integration of basic services towards the value of creating use cases.
 
-> **Warning**
-> Software / Hardware prerequisites needed here
+The basecamp is maintained by the IDSA to ensure its alignment with the IDSA reference architecture and specifications; V 1.0 of the basecamp consists of the following IDS based infrastructure components: CA, DAPS, ParIS, Metadata Broker, Transaction Log (Clearing House). V2.0 will add App Store and Vocabulary Provider.
```

Comment:
Maintenace is not provided by IDSA but by the community. IDSA is hosting this repository.